Backend/app.py

```diff
@@ -15,4 +15,16 @@ def parts():
 
 @app.route("/api/parts", methods=["POST"])
 def newParts():
-    return "Hello!"
\ No newline at end of file
+    print(request.form)
+    return "Hello!"
+
+
+@app.route('/uploader', methods = ['GET', 'POST'])
+def upload_file():
+   if request.method == 'POST':
+      f = request.files['my_file']
+      f.save(secure_filename(f.filename))

```

Comment:
We will probably need to add the following line to import this function: 
```
from werkzeug.utils import secure_filename
```

Reply:
This can be added between lines 1 and 2 (right after the existing import, but before `app = Flask(__name__)`